sdk/helper/consts/consts.go

```diff
@@ -19,6 +19,10 @@ const (
 	// SSRF protection.
 	RequestHeaderName = "X-Vault-Request"
 
+	// WrapTTLHeaderName is the name of the header containing a directive to
+	// wrap the response
+	WrapTTLHeaderName = "X-Vault-Wrap-Ttl"
```

Comment:
What do you think about  using `X-Vault-Wrap-TTL` instead?

Reply:
Done in https://github.com/hashicorp/vault/pull/23924/commits/195d82caa6d8ee09aeeb6d95cbc3228b9c330746